src/Peering/Peering/Peering/GetAzurePeeringReceivedRouteCommand.cs

```diff
@@ -0,0 +1,152 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+namespace Microsoft.Azure.PowerShell.Cmdlets.Peering.Peering
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Management.Automation;
+
+    using Microsoft.Azure.Commands.Peering.Properties;
+    using Microsoft.Azure.Management.Internal.Resources.Utilities.Models;
+    using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+    using Microsoft.Azure.Management.Peering;
+    using Microsoft.Azure.Management.Peering.Models;
+    using Microsoft.Azure.PowerShell.Cmdlets.Peering.Common;
+    using Microsoft.Azure.PowerShell.Cmdlets.Peering.Models;
+
+    /// <inheritdoc />
+    /// <summary>
+    ///     The Get Az InputObject cmdlet.
+    /// </summary>
+    [Cmdlet(VerbsCommon.Get, Constants.AzPeeringReceivedRoutes,
+        DefaultParameterSetName = Constants.ParameterSetNameByResourceAndName)]
+    [OutputType(typeof(List<PSPeeringReceivedRoute>))]
```

Comment:
the return type should be "PSPeeringReceivedRoute" instead of "List<PSPeeringReceivedRoute>"

Reply:
Hi @rileymckenna 

Can you fix this and regenerate help markdown for it?